### PR TITLE
Zs flakeyexplorertest16587

### LINKED ISSUE #16587: Flakey explorer test

### PR DESCRIPTION
**Pull request by zoesteinkamp**
Closes #16587 

Describe your proposed changes here.

- [ ] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [ ] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [ ] Rebased/mergeable
- [ ] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [ ] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: ui/cypress/e2e/explorer.test.ts [hunk 1]
```diff
@@ -357,15 +357,18 @@ describe('DataExplorer', () => {
 
   describe('raw script editing', () => {
     beforeEach(() => {
-      cy.getByTestID('switch-to-script-editor').click()
+      cy.getByTestID('switch-to-script-editor')
+        .should('be.visible')
+        .click()
     })
 
     // TODO: fix flakeyness of this test
```
Comment (drdelambre) [inline ui/cypress/e2e/explorer.test.ts]: can kill this line?

File: ui/cypress/e2e/explorer.test.ts [hunk 2]
```diff
@@ -37,7 +37,7 @@ describe('DataExplorer', () => {
 
     cy.fixture('routes').then(({orgs, explorer}) => {
       cy.get<Organization>('@org').then(({id}) => {
-        cy.visit(`${orgs}/${id}${explorer}`)
+        cy.visit(`${orgs}/${id}${explorer}`, {timeout: 3000})
```
Comment (drdelambre) [inline ui/cypress/e2e/explorer.test.ts]: it's better to do a combo of `cy.visit` and then use a `cy.get` to see if things have loaded

File: ui/cypress/e2e/explorer.test.ts (hunk 1 above)
Comment (zoesteinkamp) [reply ui/cypress/e2e/explorer.test.ts]: killed in cold blood

File: ui/cypress/e2e/explorer.test.ts (hunk 2 above)
Comment (zoesteinkamp) [reply ui/cypress/e2e/explorer.test.ts]: Im going to remove the timeout, i think we dont need it, i just needed it for local testing

**Pull request merged**